src/settings/LocationLocations/LocationForm/RemoteStorageField.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useField } from 'react-final-form';
+
+import { Control, useRemoteStorageApi } from '../RemoteStorage';
+
+export const RemoteStorageField = ({ parentMutator, initialValues, checkLocationHasHoldingsOrItems }) => {
+  const { remoteMap, mappings, t } = useRemoteStorageApi();
+
+  const [readOnly, setReadOnly] = React.useState(true);
```

Comment:
Since it's boolean value, please use appropriate naming.
```suggestion
  const [isReadOnly, setIsReadOnly] = React.useState(true);
```